code/game/objects/structures/signs.dm

```diff
@@ -241,4 +241,42 @@
 /obj/structure/sign/crime
 	name = "CRIME DOES NOT PAY"
 	desc = "A warning sign which suggests that you reconsider your poor life choices."
-	icon_state = "crime"
\ No newline at end of file
+	icon_state = "crime"
+
+/obj/structure/sign/chinese
+	name = "Imcomprehensible Sign"
```

Comment:
You're implying I actually proof read code instead of nitpicking on low hanging fruit. Hah!